pkgs/development/ocaml-modules/linol/default.nix

```diff
@@ -1,22 +1,15 @@
-{ lib, fetchFromGitHub, fetchpatch, buildDunePackage, yojson, logs, lsp, ppx_yojson_conv_lib, trace }:
+{ lib, fetchurl, buildDunePackage, yojson, logs, lsp, ppx_yojson_conv_lib, trace }:
 
 buildDunePackage
 rec {
   pname = "linol";
-  version = "0.5";
+  version = "0.6";
 
   minimalOCamlVersion = "4.14";
 
-  src = fetchFromGitHub {
-    owner = "c-cube";
-    repo = "linol";
-    rev = "v${version}";
-    hash = "sha256-ULPOB/hb+2VXDB/eK66WDDh/wj0bOwUt0tZsiIXqndo=";
-  };
-
-  patches = fetchpatch {
-    url = "https://github.com/c-cube/linol/commit/d8ebcf9a60f1e7251d14cdcd0b2ebd5b7f8eec6d.patch";
-    hash = "sha256-JHR0P0X3ep5HvDWW43dMb452/WsFKS4e+5Qhk4MzaxQ=";
+  src = fetchurl {
```

Comment:
Is there a reason you switched from Github source to the download artefact? (If so, a comment or a note in the commit message would be appreciated)